fix error message for when the broker config file is not supplied

Right now, if the broker config file is not supplied, running the broker results in the following panic:

```
panic: runtime error: index out of range

goroutine 1 [running]:
main.main()
        /Users/nimakaviani/go/src/github.com/cloudfoundry-incubator/blockhead/cmd/broker/main.go:17 +0x41c
```

We should do better validation and have proper error message when the issue is with a missing config file.

--- a/cmd/broker/main.go
+++ b/cmd/broker/main.go
@@ -1,7 +1,6 @@
 package main
 
 import (
-	"fmt"
 	"net/http"
 	"os"
 
@@ -14,9 +13,13 @@ import (
 
 func main() {
 	logger := lager.NewLogger("blockhead-broker")
-	configFilepath := os.Args[1]
 	logger.RegisterSink(lager.NewWriterSink(os.Stdout, lager.DEBUG))
-	logger.Debug(fmt.Sprintf("reading config file %q", configFilepath))
+
+	if len(os.Args) < 2 {
+		panic("config file missing")
+	}
+
+	configFilepath := os.Args[1]
 	cfg, err := config.NewConfig(configFilepath)
 	if err != nil {
 		panic(err)

